packages/terra-filter-view/src/_OptGroup.module.scss

```diff
@@ -0,0 +1,32 @@
+:local {
+  .opt-group {
+    list-style: none;
+    margin: var(--terra-select-optgroup-margin, 0);
+    padding: var(--terra-select-optgroup-padding, 0);
+    user-select: none;
+
+    &:not(:first-child) > .label {
+      border-top: var(--terra-select-optgroup-label-border-top, 0.1428571429rem solid #dedfe0);
+    }
+  }
+
+  .label {
+    background-color: var(--terra-select-optgroup-label-background-color, #f4f4f4);
+    color: var(--terra-select-optgroup-label-color, #64696c);
+    font-size: var(--terra-select-optgroup-font-size, 0.9286rem);
```

Comment:
When these are namespaced this one should also be updated to include the `label` prefix